README.md

```diff
@@ -2,7 +2,7 @@
 
 [![NPM version](https://img.shields.io/npm/v/@brightspace-ui/testing.svg)](https://www.npmjs.org/package/@brightspace-ui/testing)
 
-Testing utilities that build upon [@web/test-runner](https://modern-web.dev/docs/test-runner/overview/) but are specifically designed and configured to work with Brightspace components and applications.
+Testing utilities which are specifically designed and configured for Brightspace UI components and applications.
```

Comment:
I don't want to take away that this is building on other stuff, but I moved this information down to where it's most relevant.